.changeset/small-brooms-destroy.md

```diff
@@ -0,0 +1,7 @@
+---
+"@ilo-org/react": patch
+"@ilo-org/styles": patch
+"@ilo-org/twig": patch
+---
+
+Address accessibility issues with heading order in cards by changing `<h3>` tags to `<p>`, Additionaly add an optional prop that allows for configuration of the title element if needed.
```

Comment:
I would change this to:

Address accessibility issues with the heading order of cards by changing the default element used to render the title from an `<h3>` to `<p>`. Additionaly add an optional prop that allows for configuration of the title element if needed.